### DISCUSSION TITLE
Is there an option to change the number of repeats per image

### DISCUSSION THREAD

**Discussion by TrophiHunter** [d8ahazard/sd_dreambooth_extension#336] [General]
I come from the Dreambooth-SD-optimized repo which uses v1-finetune_unfrozen.yaml, it has a repeats option, is there an equivalent setting here in the ui?

**Comment by d8ahazard** [comment]
I've added an option to stop training a specific concept after a number of steps - the training wizards will auto-calculate this for you based on the number of instance images.
